installing/installing_bare_metal_ipi/ipi-install-overview.adoc

```diff
@@ -4,19 +4,25 @@
 include::_attributes/common-attributes.adoc[]
 :context: ipi-install
 
-Installer-provisioned installation provides support for installing {product-title} on bare metal nodes. This guide provides a methodology to achieving a successful installation.
-
-During installer-provisioned installation on bare metal, the installation program on the bare metal node labeled as `provisioner` creates a bootstrap virtual machine (VM). The role of the bootstrap VM is to assist in the process of deploying an {product-title} cluster. The bootstrap VM connects to the `baremetal` network and to the `provisioning` network, if present, via the network bridges.
+Installer-provisioned installation on bare metal nodes deploys and configures the infrastructure that a {product-title} cluster runs on. This guide provides a methodology to achieving a successful installer-provisioned bare-metal installation. The following diagram illustrates the installation environment in phase 1 of deployment.
 
 image::210_OpenShift_Baremetal_IPI_Deployment_updates_0122_1.png[Deployment phase one]
 
-The provisioning node can be removed after the installation.
+For the installation, the key elements in the above diagram are:
```

Comment:
Per ISG:
Do not use to refer to a subsequent software version or fix level. Use "later". 2. Do not use to indicate a relative location in a document, as in "the above restrictions" or "the steps above". Use "previous" or "preceding". Examples: "The previous restrictions apply to ..."; "The preceding steps..."

Suggest
"For the installation, the key elements in the previous diagram are as follows:"

Reply:
Done. Thanks for the edits, Asleigh!